torchmultimodal/models/vqvae.py

```diff
@@ -52,18 +53,30 @@ def __init__(
         self.decoder = decoder
         self.codebook = Codebook(codebook_num_embeddings, codebook_embedding_dim)
 
-    def encode(self, x: Tensor) -> CodebookOutput:
-        return self.codebook(self.encoder(x))
+    def latent_shape(self, input_shape: Union[Size, Tuple]) -> Tuple[int, ...]:
```

Comment:
it would be good to add a quick comment saying this is the shape of the encoder output (flattened?)

Reply:
Added. No it's not flattened, only downsampled in each of the dims (d1,...,dn)